crates/y-sweet-core/src/auth.rs

```diff
@@ -1,18 +1,56 @@
-use std::fmt::Display;
-
-use base64::{
-    alphabet::{STANDARD, URL_SAFE},
-    engine::{DecodePaddingMode, GeneralPurpose, GeneralPurposeConfig},
-    Engine,
-};
 use bincode::Options;
+use data_encoding::Encoding;
 use rand::Rng;
 use serde::{Deserialize, Serialize};
 use sha2::{Digest, Sha256};
+use std::fmt::Display;
 use thiserror::Error;
 
 const EXPIRATION_MILLIS: u64 = 1000 * 60 * 60; // 60 minutes
 
+/// Static version of:
+/// 
+/// ```
+/// use data_encoding::{Specification, BASE64URL_NOPAD, Translate};
+/// let spec = Specification {
+///     ignore: "=".to_string(),
+///     translate: Translate {
+///         from: "/+".to_string(),
+///         to: "_-".to_string(),
+///     },
+///     ..BASE64URL_NOPAD.specification()
+/// };
+/// use y_sweet_core::auth::BASE64_CUSTOM;
+/// assert_eq!(BASE64_CUSTOM, spec.encoding().unwrap());
```

Comment:
This assert is tested by `cargo test`, which ensures that if the code above is updated, the encoding below needs to be too.

Reply:
Oh interesting! Would it be too much of a hassle to include this explanation as a comment in the code?